### PR TITLE
Basic playlist support

### LINKED ISSUE #57: Playlist file support
Opening a .pls file currently does not work -- would be quite a useful feature.

### PR DESCRIPTION
**Pull request by sfan5**
not <i>that</i> basic actually
fixes #57

UI:
screenshot_20171208-233157

### CODE REVIEW

**Comment by sfan5** [comment]
@Veloxigami if you want to try: https://kitsunemimi.pw/tmp/mpv-android-2017-12-08.apk
`.pls` files are supported, but won't open directly from the browser (use "Open URL")

File: app/src/main/res/layout/player.xml [hunk 1]
```diff
@@ -86,8 +106,6 @@
                 android:layout_height="wrap_content"
                 android:text=".."
                 android:textColor="@android:color/white"
-                android:src="@drawable/ic_short_text_black_24dp"
-                android:tint="#ffffff"
```
Comment (sfan5) [inline app/src/main/res/layout/player.xml]: (this was unused because this is not an `ImageButton`)

**Comment by qqgg231** [comment]
@sfan5 great music taste ^_^

It is not possible to open pls file via file manager (it only shortly blinks), but possible to open via mpv. Is it by design?  Is it possible to fix this?

**Comment by sfan5** [comment]
@qqgg231 when you open via file manager, mpv gets a `content://` URI
mpv-android used to try to resolve it back to the file path, but it no longer does

**Pull request merged**